doc/sphinx/source/projects/ad2s1210/ad2s1210.rst

```diff
@@ -0,0 +1,105 @@
+AD2S1210 IIO Application
```

Comment:
Any reason why this file is named as "as2s1210.rst" and not "ad2s1210_iio.rst" ?
Should we maintain a common file naming style across the repo for all project specific RST files?

Reply:
Sorry- typo on my end. Read as "a**d**2s1210.rst" and not "ad2s1210_iio.rst" ?